README.md

```diff
@@ -1,2 +1,10 @@
-# Hello-Microverse
-Linter Testcase
+# Hello Microverse
+
+> Apply linter testcase on HTML/CSS.
+
+
+## Built With
+
+- HTML
+- CSS
+- Linters: Stylelint, WebHint, LightHouse
```

Comment:
- [ ] Please make sure you add all necessary sections to your `README` file suggested by [Microverse](https://github.com/microverseinc/readme-template)🙏 .  Like the 

> **Author**
> **Contributing**
> **Show your support**
> **Acknowledgement and** 
> **License**

Reply:
I have changed the readme file accordingly.